pages/api/submit.ts

```diff
@@ -1,22 +1,33 @@
 import { NotifyClient } from "notifications-node-client";
 import type { NextApiRequest, NextApiResponse } from "next";
+import {
+  SubmissionRequestBody,
+  SubmissionParsedRequest,
+  FileInputResponse,
+  ProcessedFile,
+  PublicFormSchemaProperties,
+  Response,
+  Responses,
+} from "@lib/types";
 import { LambdaClient, InvokeCommand } from "@aws-sdk/client-lambda";
-import formidable, { Fields, Files } from "formidable";
 import convertMessage from "@lib/markdown";
 import { rehydrateFormResponses } from "@lib/integration/helpers";
 import { getFormByID, getSubmissionByID } from "@lib/integration/crud";
 import { logMessage } from "@lib/logger";
-import { PublicFormSchemaProperties, Responses } from "@lib/types";
 import { checkOne } from "@lib/flags";
 import { pushFileToS3, deleteObject } from "@lib/s3-upload";
-import fs from "fs";
+import { fileTypeFromBuffer } from "file-type";
+import { Readable } from "stream";
 
 export const config = {
   api: {
     bodyParser: false,
   },
 };
 
+export const acceptedFileMimeTypes =
```

Comment:
This will go a long way to preventing invalid file types from being submitted! Nice stuff. LGTM from a file security perspective. Minor UX suggestion below.

If it doesn't exist already, i'd also recommend having a client side file extension validation in addition to this while also displaying accepted file types in the client views.

This will allow regular users to get immediate feedback and potentially communicate with the form owner that they'd like to submit a doc that's currently not allowed.